03/13/2021 06:09:27 AM : logging_configuration.logging_config : 
Traceback (most recent call last):
  File "D:\Pycharm Workshop\ProblemsSolvedUsingPython\basic_core_programs\power_of_2.py", line 29, in <module>
    number = int(argv[1])
IndexError: list index out of range
03/13/2021 06:16:31 AM : logging_configuration.logging_config : 
Traceback (most recent call last):
  File "D:\Pycharm Workshop\ProblemsSolvedUsingPython\basic_core_programs\power_of_2.py", line 30, in <module>
    if 0 <= number < 31:
TypeError: '<=' not supported between instances of 'int' and 'str'
